mastodon4j/src/main/java/com/sys1yagi/mastodon4j/api/exception/Mastodon4jRequestException.kt

```diff
@@ -2,4 +2,13 @@ package com.sys1yagi.mastodon4j.api.exception
 
 import okhttp3.Response
 
-class Mastodon4jRequestException(val response: Response) : Exception(response.message())
+class Mastodon4jRequestException : Exception {
+    val response: Response?
+
+    constructor(response: Response) : super(response.message()) {
+        this.response = response
+    }
+    constructor(e : Exception) : super(e) {
+        this.response = null
+    }
```

Comment:
IMO: I want to the function of distinguish whether it is an error response or not. like below.

```kotlin
fun isErrorResponse() = response != null
```